pkg/controllers/csi/metadata/correctness_test.go

```diff
@@ -9,39 +9,46 @@ import (
 	"github.com/Dynatrace/dynatrace-operator/pkg/api/scheme/fake"
 	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta1/dynakube"
 	dtcsi "github.com/Dynatrace/dynatrace-operator/pkg/controllers/csi"
+	testutil "github.com/Dynatrace/dynatrace-operator/pkg/util/testing"
+	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"gorm.io/gorm"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func createTestDynakube(index int) Dynakube {
-	return Dynakube{
-		TenantUUID:             fmt.Sprintf("asc%d", index),
-		LatestVersion:          strconv.Itoa(123 * index),
-		Name:                   fmt.Sprintf("dk%d", index),
-		ImageDigest:            fmt.Sprintf("sha256:%d", 123*index),
-		MaxFailedMountAttempts: index,
+func createTestTenantConfig(index int) *TenantConfig {
+	return &TenantConfig{
+		Name:                        fmt.Sprintf("dk%d", index),
+		TenantUUID:                  fmt.Sprintf("asc%d", index),
+		DownloadedCodeModuleVersion: fmt.Sprintf("sha256:%d", 123*index),
```

Comment:
```suggestion
		DownloadedCodeModuleVersion: fmt.Sprintf("v%d", 123*index),
```

Reply:
don't want to confuse because of image sha256

Reply:
Done.
I even removed the `v` because we don't store it with `v` in the database.